cups/templates/printers.conf.j2

```diff
@@ -0,0 +1,22 @@
+# Printer configuration file for CUPS v1.7.2
+# Written by cupsd
+# DO NOT EDIT THIS FILE WHEN CUPSD IS RUNNING
+<DefaultPrinter contest>
+UUID urn:uuid:db002b1c-c421-3c4b-46da-d0e7d4d2ebb4
+AuthInfoRequired none
+Info contest
+DeviceURI ipp://printsvr:631/classes/contest
```

Comment:
Maybe a typo?
```suggestion
DeviceURI ipp://printsrv:631/classes/contest
```

Reply:
no printsrv is the correct,  lastminute2022.yml was creating the wrong name.